content/plugins/notes/travis-ci.md

```diff
@@ -2,25 +2,17 @@
 name: travis-ci
 ---
 
-To use the Travis CI Backstage plugin, first visit Travis CI to get an API token.
+## Features
 
-There are two versions of Travis CI: https://travis-ci.com and https://travis-ci.org. Tokens
-from .com do not work for .org and vice versa, so make sure you are on the right Travis website.
+- List Travis CI Builds
+- Retrigger builds
 
-Tokens can be copied from the settings page in Travis CI. Use the menu in the top right of the page to navigate to Settings. Click Settings again, under your name this time, and you should see API authentication.
+To use the Travis CI Backstage plugin, first visit Travis CI to get an API token: https://travis-ci.com/account/preferences
```

Comment:
I know we're pitching at a technical audience but it's worth being very specific about how they issue a token. A link to Travis CI docs is fine if there is one or you can write a few lines. The user will ignore it if they know what they're doing.

Reply:
This link points to the settings page shown in the image, on which there is a first form field "API KEY" which has to be copied. This assumes that the user has already created an account.